funding/manager.go

```diff
@@ -2458,18 +2458,14 @@ func (f *Manager) fundingTimeout(c *channeldb.OpenChannel,
 	go func() {
 		defer f.wg.Done()
 
-		peerChan := make(chan lnpeer.Peer, 1)
-		var peerKey [33]byte
-		copy(peerKey[:], c.IdentityPub.SerializeCompressed())
-
-		f.cfg.NotifyWhenOnline(peerKey, peerChan)
-
-		var peer lnpeer.Peer
-		select {
-		case peer = <-peerChan:
-		case <-f.quit:
+		peer, err := f.waitForPeerOnline(c.IdentityPub)
+		if err != nil {
+			// waitForPeerOnline only returns an error if the
```

Comment:
nit: I think it's better to check against `ErrFundingManagerShuttingDown` explicitly so we don't need to "remember" that `waitForPeerOnline` only returns this error.